packages/plugins/typescript/resolvers/src/index.ts

```diff
@@ -186,8 +208,25 @@ export type DirectiveResolverFn<TResult = {}, TParent = {}, TContext = {}, TArgs
     console['warn'](`Unused mappers: ${unusedMappers.join(',')}`);
   }
 
+  const prepend: string[] = [];
+
+  if (imports.length) {
+    prepend.push(`import { ${imports.join(', ')} } from 'graphql';`);
+  }
+
+  if (config.customResolveInfo) {
+    const [importName, moduleName] = config.customResolveInfo.split('#');
```

Comment:
I think it's better to use here `parseMapper` method, it will handle the `#` but also other use cases.